src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Models/RazorSemanticTokensLegend.cs

```diff
@@ -10,6 +10,8 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Semantic.Models
 {
     internal class RazorSemanticTokensLegend
     {
+        private const string RazorComponentElementString = "razorComponentElement";
+        private const string RazorComponentAttributeString = "RazorComponentAttribute";
```

Comment:
Out of curiosity why is this particular case pascal-cased while the other strings are camel-cased?

Reply:
Basically I just have to [match this line](https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient?path=%2Fsrc%2Fproduct%2FRemoteLanguage%2FImpl%2FFeatures%2FSemanticTokens%2FTagger%2FSemanticTokensTaggerBase.cs&version=GBdevelop&line=473&lineEnd=474&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents) which happens to be cased that way (probably due to a typo. I intend to send a PR to VSLanguageServerClient to  make it accept both versions before changing things here and then removing the `R` variant, but I figured the rest of this coloration work didn't need to wait on that since it's a behind the scenes detail.